static/_redirects

```diff
@@ -14,7 +14,8 @@
 /ko/docs/     /ko/docs/home/ 301!
 /no/docs/     /no/docs/home/ 301!
 /pl/docs/     /pl/docs/home/ 301!
-/pt/docs/     /pt/docs/home/ 301!
+/pt/*         /pt-br/:splat  302!
```

Comment:
Please move this outside the redirects for `/docs/` to `/docs/home` - it belongs elsewhere really.